frontend/src/scenes/dashboard/dashboardLogic.ts

```diff
@@ -342,15 +343,14 @@ export const dashboardLogic = kea<dashboardLogicType<DashboardLogicProps>>({
                 if (!items || !items.length) {
                     return null
                 }
-                let lastRefreshed = items[0].last_refresh
-
+                let oldestLastRefreshed = items[0].last_refresh ? dayjs(items[0].last_refresh) : null
```

Comment:
This could just be `= null` since the same value is calculated in the loop below.

Reply:
Good call - updated